c2corg_ui/templates/image/index.html

```diff
@@ -7,9 +7,9 @@
 <%block name="moduleConstantsValues">
   module.value('mapFeatureCollection', null);
   module.value('advancedSearchFilters', {
-    'date': {'type': 'date'},
     'act': {'type': 'list'},
-    'oalt': {'type': 'range'}
+    'cat': {'type': 'list'},
```

Comment:
There is no ``oalt`` filter for images, it's probably an incorrect copy-paste of the outings filters.
Here are the available filters for images: https://github.com/c2corg/v6_api/blob/master/c2corg_api/search/mappings/image_mapping.py#L11-L19

I have added some filters that were not correctly defined. Those definitions are used in the search filters directive, for instance to parse the filtering criterias in the URL: https://github.com/c2corg/v6_ui/blob/a86893997e2b92976d79dab55b15c467eedad7ac/c2corg_ui/static/js/search/searchfilters.js#L163-L193

Currently when loading https://www.camptocamp.org/images#bbox=773697%252C5610223%252C778464%252C5617799&cat=action%252Ctrack&act=snow_ice_mixed%252Cmountain_climbing
* activities are correctly restored (because ``act`` is defined in the list above)
* categories are not (see for instance the number of selected items in the categories list)
![capture du 2017-06-21 13-10-30](https://user-images.githubusercontent.com/1192331/27381451-34c703fc-5683-11e7-92e2-3e9d16370354.png)